requirements/edx/github.txt

```diff
@@ -78,7 +78,7 @@ git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
 git+https://github.com/edx/edx-ora2.git@1.4.0#egg=ora2==1.4.0
 -e git+https://github.com/edx/edx-submissions.git@2.0.0#egg=edx-submissions==2.0.0
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
-git+https://github.com/edx/edx-val.git@0.0.13#egg=edxval==0.0.13
+git+https://github.com/edx/edx-val.git@c4929c6a56d39c6cf462e081e291026d6747595c#egg=edxval==c4929c6a56d39c6cf462e081e291026d6747595c
```

Comment:
Reminder: would need to add val release tags instead of commits before squashing commits :)